acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/tests/actions/JakartaTaskAcceptanceTest.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.acceptance.tests.actions
+
+import com.chutneytesting.acceptance.common.*
+import com.chutneytesting.kotlin.dsl.ChutneyScenario
+import com.chutneytesting.kotlin.dsl.DebugAction
+import com.chutneytesting.kotlin.dsl.JakartaBrokerStartAction
+import com.chutneytesting.kotlin.dsl.Scenario
+
+
+fun `Jakarta actionInputs wrong url`(
+  jakartaAction: String,
+  actionInputs: String
+): ChutneyScenario {
+  return Scenario(title = "Jakarta actionInputs wrong url") {
+    val environmentName = "JAK_$jakartaAction"
+    Given("A target pointing to an unknown service") {
+      createEnvironment(
+        environmentName,
+        """
+        [
+          {
+              "name": "test_jakarta",
+              "url": "tcp://localhost:11111",
```

Comment:
Maybe use always the same url for target describing an unknown service, as `tcp://unknown.host;1234`.
Same in Jms and Http. Ok in Kafka.